HealthcareSystem/IntegrationAdapters/Controllers/PharmacyController.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using Backend.Model.Pharmacies;
+using Backend.Repository;
+using Backend.Service.Pharmacies;
+using Microsoft.AspNetCore.Mvc;
+
+namespace IntegrationAdapters.Controllers
+{
+    public class PharmacyController : Microsoft.AspNetCore.Mvc.Controller
+    {
+        private readonly PharmacyService _pharmacyService;
+
+        public PharmacyController(IPharmacyRepo iPharmacyRepo)
+        {
+            _pharmacyService = new PharmacyService(iPharmacyRepo);
```

Comment:
There's an IPharmacyService interface, why inject the repo and instantiate the service here instead of just injecting the service? It's already added as scoped in the startup as well, was this an oversight or were there some issues with DI?

Reply:
Yeah it was an oversight on my part. Thanks for pointing it out, I just pushed a fix for it

Reply:
Oversight, no serious issues. Fixed in 0dd5b746d447e1725aac9e50581071ff190decad.